crates/stark-backend/src/chip.rs

```diff
@@ -37,6 +37,12 @@ pub trait ChipUsageGetter {
     }
 }
 
+/// A chip whose state could be persisted.
+pub trait PersistStateChip<S> {
```

Comment:
I thought you wanted to also implement the trait on some non-chips like `VmChipComplex`? so `Chip` doesn't need to be in the trait name